### ISSUE TITLE
Legend coverage/diff colors

### ISSUE THREAD

**Issue by Flamefire** [codecov/support#183]
I find the diffs hard to read as you have green and red and yellow colors on the lines which are sometimes mixed. I did not found a legend or explanation on how to read them.

E.g. https://codecov.io/gh/Return-To-The-Roots/s25client/compare/e850e5731a977988ea879bb613330b93071e407c...d64200b8baac9b54b1ac1f819511d935055a245c?path=src%2Fdrivers%2FDriverWrapper.cpp&before=src%2Fdrivers%2FDriverWrapper.cpp#7372632F6D61696E2E637070-289

I assume light red/green = source diff; dark red/green = coverage. But what is the mixed one trying to say (highlighted line)?

**Comment by stevepeak** [comment]
@Flamefire Nice suggestion :) I'll look into adding a legend of sorts

**Comment by BenjamenMeyer** [comment]
Yes...trying to understand how codecov doesn't think some code is covered in https://codecov.io/gh/rackerlabs/mimic/compare/b7cb2b2c480170a68ae61cae200f3801463b3fb8...d9bb2832e24a1d658f88af3d3e0ac768667695eb#6D696D69632F726573742F6964656E746974795F6170692E7079-899 and a legend to understand the yellow (in that particular line) or the (in lower lines) the yellow/red would be extremely helpful.

**Comment by stevepeak** [comment]
@BenjamenMeyer which line(s) specifically. This diff looks good upon my first glance. The light green/red is the diff add/remove. The darker green/yellow/red are coverage metrics on top of the diff. Thank you

**Comment by BenjamenMeyer** [comment]
@stevepeak Line 899 specifically shows a yellow. It was actually a missed branch in coverage. Likewise 975-976, and 1003-1004 are due to coverage. However, differentiating between them is not easy.

Also, using the same colors for diff and coverage but slightly different shades is not the best thing to do. The two should use completely separate color pallets especially since some monitors may render the shades close together and make differentiation hard.

Example - all the lines I mentioned above are due to coverage (which I know because I was able to resolve via other tools); however, I can only see a difference in shades on line 899 which seems to have a yellow column and orange-yellow numeric column. Neither of these "columns" are labeled either, no documentation provided, etc. Further, all the lines references are *new* in the related GitHub PR, therefore only Greed would be expected.
